Automattic/wp-calypso
# Add credentials interface to Settings/Security

This PR is the beginning of the credentials interface for Jetpack Backups.

Fixes https://github.com/Automattic/wp-calypso/issues/18831

**Testing instructions:**
1. Select a self-hosted Jetpack site with a plan, navigate to settings/security, and attempt to add valid SSH, SFTP, or FTP credentials to the form. 
2. Ensure the credentials are present, and the site is connected, after refreshing and coming back to the form.
3. Ensure that credentials can be updated by resubmitting the form.
